src/rdm/param_data.c

```diff
@@ -0,0 +1,794 @@
+/******************************************************************************
+ * Copyright 2019 ETC Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************
+ * This file is a part of RDM. For more information, go to:
+ * https://github.com/ETCLabs/RDM
+ ******************************************************************************/
+#include "rdm/param_data.h"
+
+// e1.33 Table 7-1: Required E1.20 Parameter IDs for the Default Responder
+// Get/Set IDENTIFY_DEVICE See Section 10.11.1 of [RDM]
+etcpal_error_t rdmpd_unpack_get_resp_identify_device(const RdmParamData *pd, bool *identify_state_out)
```

Comment:
Write doc comments for each of these functions as you implement them. See the existing functions in the other modules for reference.